packages/diagrams/lib/src/box_decoration.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+const String _boxDecoration = 'box_decoration';
+
+class BoxDecorationDiagram extends StatelessWidget implements DiagramMetadata {
+  const BoxDecorationDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    return ConstrainedBox(
+      key: UniqueKey(),
+      constraints: BoxConstraints.tight(const Size(300, 200)),
+      child: Container(
+        alignment: FractionalOffset.center,
+        padding: const EdgeInsets.all(8),
+        color: Colors.white,
+        child: Container(
+          decoration: BoxDecoration(
+            color: const Color(0xff7c94b6),
+            image: DecorationImage(
+              image: const AssetImage('assets/owl-2.jpg'),
```

Comment:
Nit: This could be accessed using network image and referring to the image kept here on GitHub. That would prevent us from duplicating assets in different directories of this repo.

Reply:
Good call!